### PR TITLE
Added POA support

### PR DESCRIPTION
**Pull request by chinmay-browserstack**
Implemented POA support for appium python SDK.

### CODE REVIEW

File: percy/lib/percy_automate.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            return None
+        if not isinstance(name, str):
+            raise TypeError('Argument name should be a string')
+
+        try:
+            ignore_region_elements = [element.id for element in kwargs.get(IGNORE_ELEMENT_KEY, [])]
+            kwargs.pop(IGNORE_ELEMENT_KEY, None)
+
+            # Post to automateScreenshot endpoint with driver options and other info
+            response = requests.post(f'{PERCY_CLI_API}/percy/automateScreenshot', json={
```
Comment (pankaj443) [inline percy/lib/percy_automate.py:33]: this should be under CLI Wrapper and update similar to how we do for `post_screenshots`

File: percy/lib/percy_automate.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                'options': { **kwargs, "ignore_region_elements": ignore_region_elements }
+            }, timeout=30)
+
+            # Handle errors
+            response.raise_for_status()
+            data = response.json()
+            if not data['success']: raise CLIException(data['error'])
+        except Exception as e:
+            log(f'Could not take Screenshot "{name}"')
+            log(f'{e}')
```
Comment (pankaj443) [inline percy/lib/percy_automate.py:50]: log complete on debug

File: percy/lib/cli_wrapper.py [hunk 3]
```diff
@@ -31,7 +31,7 @@ def is_percy_enabled():
                 log(f'Unsupported Percy CLI version, {version}')
                 return False
 
-            return True
+            return data['type']
```
Comment (pankaj443) [inline percy/lib/cli_wrapper.py:34]: is_percy_enabled() should ideally true/false only depending on status. you can set another static var similar to `Environment.percy_build_id`  for type and use that for your use case.

File: percy/lib/percy_automate.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import os
+from appium.webdriver.webdriver import WebDriver
+
+from percy.common import log
+from percy.errors import DriverNotSupported
+from percy.lib.percy_options import PercyOptions
+from percy.lib.cli_wrapper import CLIWrapper
+
+# Maybe get the CLI API address from the environment
+PERCY_CLI_API = os.environ.get('PERCY_CLI_API') or 'http://localhost:5338'
```
Comment (pankaj443) [inline percy/lib/percy_automate.py:10]: this is not required in this file anymore

**Review by pankaj443 — DISMISSED** [review]
LGTM, please do sanity checks before merging

**Review by Amit3200 — DISMISSED** [review]

**Review by pankaj443 — APPROVED** [review]

**Review by Amit3200 — APPROVED** [review]

**Review by ninadbstack — APPROVED** [review]

**Pull request merged**